src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,97 @@
+package pages;
+
+import common.AbstractComponent;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class HomePage extends AbstractComponent {
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy (css = "button[title='Close'][aria-label='Close']")
+    private List<WebElement> popUpButton1;
+
+    @FindBy(xpath = "//button[@class='popup-newsletter__close link']")
+    private List<WebElement> popUpButton2;
+
+    @FindBy (xpath = "//button[@class='exit-popup__close link']")
+    private List<WebElement> popUpButton3;
+
+//    @FindBy (css = ".product-item__title")
+//    private List<WebElement> livingRoomSets;
+//
+    @FindBy (xpath = "//div[@class='product-item product-item--vertical 1/4--lap 1/5--desk 1/6--wide is-selected']")
+    private List<WebElement> livingRoomSets;
+
+    private By roomSets = By.cssSelector("//div[@class='product-item product-item--vertical 1/4--lap 1/5--desk 1/6--wide is-selected']");
+    private By popUp1 = By.cssSelector("button[title='Close'][aria-label='Close']");
```

Comment:
instead calling as popUp1, please give more understandable variable names for best coding practices.https://www.youtube.com/watch?v=NgPcXyNFRIc&ab_channel=SehoLim

Reply:
I change it.

Reply:
did you push? i still see popUp1